app/experimenter/experiments/models.py

```diff
@@ -668,6 +668,28 @@ def format_firefox_versions(self):
         else:
             return self.firefox_min_version
 
+    @property
+    def versions_integer_list(self):
+        min_integer = int(
+            ExperimentConstants.VERSION_REGEX.match(
+                self.firefox_min_version
+            ).group(0)
+        )
+
+        integers_list = [min_integer]
+
+        if self.firefox_max_version:
+            max_integer = int(
+                ExperimentConstants.VERSION_REGEX.match(
+                    self.firefox_max_version
+                ).group(0)
+            )
+
+            for x in range(min_integer + 1, max_integer + 1):
+                integers_list.append(x)
```

Comment:
would something like  `return list(range(min_integer, max_integer + 1)` be better than a for loop? not sure?

Reply:
Weeeeelllllllll, think about what ways it might be better?  Is the code shorter?  Is it more resilient to refactoring (ie if stuff really gets moved around and variables change)?  Does mutating variables give you an uncomfortable feeling in your gut?

Reply:
yea it seems @tiftran 's is better... because it's shorter.